roles/kubernetes-apps/network_plugin/flannel/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: "Flannel | Create ServiceAccount ClusterRole and ClusterRoleBinding"
+  command: "{{ bin_dir }}/kubectl apply -f {{ kube_config_dir }}/cni-flannel-rbac.yml"
+  run_once: true
+  when: rbac_enabled and flannel_rbac_manifest.changed
+
+#FIXME: remove when kubespray uses kubernetes 1.7 due to kubernetes/features#124
```

Comment:
why do we need to purge? we have no previous install that installed flannel as a daemonset. Lines 6-15 should be removed. We can just add proper field, by setting spec.updateStrategy.type to RollingUpdate